http_post_unsigned_tx_docs.md

```diff
@@ -0,0 +1,54 @@
+# Internal HTTP API for generating wallets and posting unsigned transactions
+
+## **Warning** only use it if you really really know what you are doing.
+
+The HTTP endpoints are only available if the miner is started with the `enable post_unsigned_tx` CLI option.
+
+## Generate a wallet and receive its secret key
+
+- **URL**
+  `/wallet`
+
+- **Method**
+  POST
+
+#### Example Response
+
+A key which can be used to sign transactions via `POST /unsigned_tx`.
+
+```javascript
+{"wallet_key":"..."}
```

Comment:
`key` in the context of wallet is misleading for me. I thought this was one of the keys in the key pair. My suggestion: `wallet_name` or `wallet_secret`

Reply:
✅